build_package.py

```diff
@@ -831,6 +831,18 @@ def build_tarball_package(variant, version, no_versioned_file_name):
     return output_name
 
 
+def replace_shebang(path, new_path, new_shebang):
+    with open(path, "r") as f:
+        with open(new_path, "w") as newf:
+            #skip shebang
+            f.readline()
+            newf.write(new_shebang)
+            newf.write("\n")
+            for line in f:
+                newf.write(line)
```

Comment:
We could probably simplify that since we know shebang is always on the first line, but this also works.

Reply:
Yeah, at least, the loop can be removed for sure :)